mission_control/navigator_missions/nav_missions/coral_survey.py

```diff
@@ -20,14 +20,22 @@ def main(navigator):
     result = navigator.fetch_result()
 
     #middle_point = np.array([-10, -70, 0]) 
-    resp = yield navigator.database_query("coral_survey")
-    if not resp.found:
-        result.success = False
-        result.response = result.DbObjectNotFound
-        result.message = "Coral survey not found in the database."
-        defer.returnValue(result)
-
-    middle_point = navigator_tools.point_to_numpy(resp.objects[0].position)
+    est_coral_survey = yield navigator.database_query("Coral_Survey")
+    assert est_coral_survey.found, "Coral Survey ROI not found."
```

Comment:
There is no need for this now.... Maybe change in the next PR. This will cause the mission to fail in the future.